Dockerfile

```diff
@@ -17,11 +17,8 @@ WORKDIR /app
 
 COPY poetry.lock pyproject.toml /app/
 
-RUN poetry config virtualenvs.create false && \
-    poetry install --no-interaction --no-ansi --no-dev --no-root
+RUN poetry install --no-interaction --no-ansi --no-dev --no-root
```

Comment:
Oh we were installing twice?
Also seems like setting virtualenvs on the container, any reason why?

Reply:
if we add this, we add an error during the installation